alfalfa_web/server/api-v2.js

```diff
@@ -110,47 +214,31 @@ router.get("/sites", async (req, res) => {
  *                 uploadTimestamp: 2022-06-30T05:43:06.885Z
  *                 uploadPath: uploads/4b8c6b40-f818-11ec-a8bd-75570e3e3a28/refrig_case_osw.zip
  *       404:
- *         description: Site ID does not exist
+ *         description: Run ID does not exist
  *         content:
  *           application/json:
  *             schema:
  *               $ref: '#/components/schemas/Error'
  */
-router.get("/sites/:siteId", async (req, res) => {
-  const { siteId } = req.params;
-
-  const error = validate(
-    { siteId },
-    {
-      siteId: "required|uuid"
-    }
-  );
-  if (error) return res.status(400).json({ error });
-
+router.get("/runs/:runId", async (req, res, next) => {
   api
-    .findSite(siteId)
-    .then((data) => {
-      if (data) {
-        return res.json({ data });
-      } else {
-        return res.status(404).json({
-          error: `Site ID '${siteId}' does not exist`
-        });
-      }
+    .formatRun(req.run)
```

Comment:
Requests to the internal api are now done with the run object instead of the id. This helps make everything more consistent and also allows us to change the id in the future without having to rewrite everything.